recipe/patches/0009-add-GRPC_DLL-mechanism-for-missing-symbols-on-window.patch

```diff
@@ -70,19 +70,39 @@ index d6ef16130a..46b067c140 100644
   public:
    typedef void (*child_postfork_func)(void);
  
+diff --git a/third_party/upb/upb/upb.c b/third_party/upb/upb/upb.c
+index 55730bb..7e6238b 100644
+--- a/third_party/upb/upb/upb.c
++++ b/third_party/upb/upb/upb.c
+@@ -112,6 +112,16 @@ static uintptr_t upb_cleanup_metadata(uint32_t* cleanup,
+   return (uintptr_t)cleanup | has_initial_block;
+ }
+ 
++// For some reason, this file does not include upb.h where the function
++// declaration with the dllexport/import spec is. Need to redefine
++// with the dllexport spec
```

Comment:
Would it not be easier to just include `upb.h` in `upb.c`?

Also, I don't think you're solving the `grpc_unsecure` case.